rules/rbe_repo.bzl

```diff
@@ -570,29 +594,53 @@ def _expand_outputs(ctx, bazel_version, project_root):
         if ctx.attr.config_dir:
             dest += ctx.attr.config_dir + "/"
         platform_dest = dest + _PLATFORM_DIR + "/"
+        java_dest = dest + _JAVA_CONFIG_DIR + "/"
+        cc_dest = dest + _CC_CONFIG_DIR + "/"
 
         # Create the directories
-        result = ctx.execute(["mkdir", "-p", "platform_dest"])
-        _print_exec_results("create output dir", result)
-
-        # Get the files that were created in the _RBE_CONFIG_DIR
-        ctx.file("local_config_files.sh", ("echo $(find ./%s -type f | sort -n)" % _RBE_CONFIG_DIR), True)
+        result = ctx.execute(["mkdir", "-p", platform_dest])
+        _print_exec_results("create platform output dir", result)
+        result = ctx.execute(["mkdir", "-p", java_dest])
+        _print_exec_results("create java output dir", result)
+        result = ctx.execute(["mkdir", "-p", cc_dest])
+        _print_exec_results("create cc output dir", result)
+
+        # Get the files that were created in the _CC_CONFIG_DIR
+        ctx.file("local_config_files.sh", ("echo $(find ./%s -type f | sort -n)" % _CC_CONFIG_DIR), True)
         result = ctx.execute(["./local_config_files.sh"])
         _print_exec_results("resolve autoconf files", result)
         autoconf_files = result.stdout.splitlines()[0].split(" ")
-        args = ["cp"] + autoconf_files + [dest]
+        args = ["cp"] + autoconf_files + [cc_dest]
+
+        # Copy the local_config_cc files to dest/{_CC_CONFIG_DIR}/
+        result = ctx.execute(args)
+        _print_exec_results("copy local_config_cc outputs", result, True, args)
 
-        # Copy the files to dest
+        # Copy the dest/{_JAVA_CONFIG_DIR}/BUILD file
+        args = ["cp", str(ctx.path(_JAVA_CONFIG_DIR + "/BUILD")), java_dest]
         result = ctx.execute(args)
-        _print_exec_results("copy outputs", result, True, args)
+        _print_exec_results("copy java_runtime BUILD", result, True, args)
 
         # Copy the dest/{_PLATFORM_DIR}/BUILD file
-        result = ctx.execute(["cp", str(ctx.path(_PLATFORM_DIR + "/BUILD")), platform_dest])
+        args = ["cp", str(ctx.path(_PLATFORM_DIR + "/BUILD")), platform_dest]
+        result = ctx.execute(args)
+        _print_exec_results("copy platform BUILD", result, True, args)
 
 # Private declaration of _rbe_autoconfig repository rule. Do not use this
 # rule directly, use rbe_autoconfig macro declared below.
 _rbe_autoconfig = repository_rule(
     attrs = {
+        "base_container_digest": attr.string(
+            doc = ("Optional. If the container to use for the RBE build " +
+                   "extends from the rbe-ubuntu16-04 image, you can " +
+                   "pass the digest (sha256 sum) of the base container here " +
+                   "and this rule will attempt use of checked-in " +
```

Comment:
"attempt to use" instead of "attempt use of"? Sounds simpler

Reply:
fixed